raven/contrib/tornado/__init__.py

```diff
@@ -11,6 +11,7 @@
 
 from tornado import ioloop
 from tornado.httpclient import AsyncHTTPClient, HTTPError
+from tornado import web
```

Comment:
Can we just do `from tornado.web import HTTPError as TornadoHTTPError` and avoid the attribute lookup?